web/includes/control_functions.php

```diff
@@ -20,9 +20,9 @@ function buildControlCommand( $monitor )
                 case 'focus' :
                 {
                     $factor = $_REQUEST['yge']/100;
-                    if ( $monitor['HasFocusSpeed'] )
```

Comment:
There are whole boat load of these changes (in this file and others).  What's the reason for them?